deployment/build-open-source-dist.sh

```diff
@@ -0,0 +1,134 @@
+#   4. Remove any unecessary artifacts from the /open-source folder (i.e. 
+#      node_modules, package-lock.json, etc.).
+#   5. Zip up the /open-source folder and create the distributable.
+#   6. Remove any temporary files used for staging.
+#
+# Parameters:
+#  - solution-name: name of the solution for consistency
+
+# Check to see if the required parameters have been provided:
+if [ -z "$1" ]; then
+    echo "Please provide the trademark approved solution name for the open source package."
+    echo "For example: ./build-open-source-dist.sh trademarked-solution-name"
+    exit 1
+fi
+
+# Get reference for all important folders
+source_template_dir="$PWD"
+dist_dir="$source_template_dir/open-source"
+source_dir="$source_template_dir/../solutions"
+github_dir="$source_template_dir/../.github"
+support_stack_dir="$source_template_dir/../deployment-prerequisties"
+
+echo "------------------------------------------------------------------------------"
+echo "[Init] Remove any old dist files from previous runs"
+echo "------------------------------------------------------------------------------"
+
+echo "rm -rf $dist_dir"
+rm -rf $dist_dir
+echo "mkdir -p $dist_dir"
+mkdir -p $dist_dir
+
+echo "------------------------------------------------------------------------------"
+echo "[Packing] GitHub templates"
+echo "------------------------------------------------------------------------------"
+
+echo "cp -r $github_dir $dist_dir"
+cp -r $github_dir $dist_dir
+
+echo "------------------------------------------------------------------------------"
+echo "[Packing] Source folder"
+echo "------------------------------------------------------------------------------"
+
+echo "cp -r $source_dir $dist_dir"
+cp -r $source_dir $dist_dir
+
+echo "------------------------------------------------------------------------------"
+echo "[Packing] cross account support folder"
+echo "------------------------------------------------------------------------------"
+
+echo "cp -r $support_stack_dir $dist_dir"
+cp -r $support_stack_dir $dist_dir
+
+
+echo "------------------------------------------------------------------------------"
+echo "[Packing] Files from the root level of the project"
+echo "------------------------------------------------------------------------------"
+
+echo "cp $source_template_dir/../LICENSE $dist_dir/LICENSE.txt"
+cp $source_template_dir/../LICENSE $dist_dir/LICENSE.txt
+
+echo "cp $source_template_dir/../NOTICE $dist_dir/NOTICE.txt"
+cp $source_template_dir/../NOTICE $dist_dir/NOTICE.txt
+
+echo "cp $source_template_dir/../README.md $dist_dir"
+cp $source_template_dir/../README.md $dist_dir
+
+echo "cp $source_template_dir/../CODE_OF_CONDUCT.md $dist_dir"
+cp $source_template_dir/../CODE_OF_CONDUCT.md $dist_dir
+
+echo "cp $source_template_dir/../CONTRIBUTING.md $dist_dir"
+cp $source_template_dir/../CONTRIBUTING.md $dist_dir
+
+echo "cp $source_template_dir/../CHANGELOG.md $dist_dir"
```

Comment:
I don't see this file in the root of our repo. Are you expecting it to be created before this step?

Reply:
Adding a sample changelog file.